README.md

```diff
@@ -57,33 +57,28 @@ including Pandas and its dependencies (numpy, etc).
 To create a new environment "myenv" with the `odm2api` package:
 
 ```
-conda create -n myenv -c odm2 python=2.7 odm2api
+conda create -n myenv -c conda-forge python=2.7 odm2api
 ```
 
-### Installing the development version from the `master` branch on github
-
-**Note from 4/26/2016:** These instructions may be slightly outdated.
-Follow these directions for installing the bleeding edge GitHub master branch,
-mainly for development and testing purposes.
-
-To create a new environment "myenv" with `odm2api`,
-first download the conda environment file
-[condaenvironment_1.yml](https://raw.githubusercontent.com/ODM2/ODM2PythonAPI/master/condaenvironment_1.yml).
-Go to the directory where `condaenvironment_1.yml` was downloaded.
-Then, on a terminal shell:
-
-```bash
-conda env create -n myenv --file py2_conda_environment.yml
-```
-
-Activate the new environment, then install `odm2api` into the environment:
-
-```bash
-activate myenv  # On Windows
-source activate myenv  # On MacOSX or Linux
-
-pip install --process-dependency-links git+https://github.com/ODM2/ODM2PythonAPI.git
-```
+### Installing the development version from the `development` branch on github
+
+Note: We follow the [Gitflow workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) for development.
+
+1. Download both `requirements.txt` and `requirements-dev.txt`.
+    ``` bash
+    wget https://raw.githubusercontent.com/ODM2/ODM2PythonAPI/master/requirements.txt
+    wget https://raw.githubusercontent.com/ODM2/ODM2PythonAPI/master/requirements-dev.txt
+    ```
+
+2. Create conda environment from the two text files.
+    ```bash
+    conda create -n odm2api_dev -c conda-forge python=2.7 --file requirements.txt --file requirements-dev.txt
+    ```
+    
+3. Install the latest commit from the development branch
+    ```bash
+    pip install git+https://github.com/ODM2/ODM2PythonAPI.git@development#egg=odm2api
```

Comment:
What does the `#egg=odm2api` part do? I'm not familiar with that syntax. I've previously pip installed from a github branch using just this:
```
pip install git+https://github.com/ODM2/ODM2PythonAPI.git@development
```

Reply:
`#egg` is just a way to explicitly state project name. Not really needed when you do just a `pip install`, but does needed if doing `pip install -e`. See [reference](https://pip.pypa.io/en/stable/reference/pip_install/#working-out-the-name-and-version).